.github/workflows/deploy.yml

```
@@ -2,38 +1,42 @@
+name: "Deploy selfhost (backend)"
+
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)